.github/workflows/release.yml

```diff
@@ -42,13 +42,10 @@ jobs:
           echo Listing upgrade directory:
           ls -lr upgrades
 
-      - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v2
-        with:
-          distribution: goreleaser
-          version: latest
-          args: release --config ../../.goreleaser.yaml --rm-dist
-          workdir: upgrades/${{ steps.getUpgradeVersion.outputs.result  }}
+      -
+        name: Run make release
+        run: make release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          UPGRADE_WORKDIR: upgrades/${{ steps.getUpgradeVersion.outputs.result  }}
           UPGRADE_VERSION: ${{ steps.getUpgradeVersion.outputs.result }}
+          GITHUB_TOKEN: ${{ secrets.GORELEASER_GITHUB_TOKEN }}
```

Comment:
Why wasn't the general github token good enough anymore?

Reply:
Good catch! Because my copy-paste-skills from the other repo was bad :D